src/pages/understand-stacks/install-api.md

```diff
@@ -24,8 +24,8 @@ $ cd blockstack-core/
 $ git checkout stacks-1.0
 $ pip install .
 $ pip install -r api/requirements.txt
-$ blockstack setup_wallet
-$ blockstack api start
+$ stx setup_wallet
```

Comment:
This is very outdated documentation and no longer relevant. We should just delete the whole thing.

Reply:
can we delete the whole file?

Reply:
These instructions are for the 1.0 blockstack-core. Either we remove the whole page, or add a notice at the top that this is out of date and remove it after 2.0 mainnet.

Reply:
added a callout